src/node.cc

```diff
@@ -950,6 +946,12 @@ int Start(int argc, char** argv) {
   return exit_code;
 }
 
+int Stop(Environment* env) {
+  CHECK(!env->is_main_thread());
+  env->GetAsyncRequest()->Stop();
+  return 0;
+}
```

Comment:
I think we want some more clearly defined semantics for this.

Do we want this to be basically a  `uv_stop()` call? You can already do that as an embedder, we don’t need to provide anything for that.

Do we want this to work like `worker.terminate()`, stopping the current JS execution – at *any* point – and not allowing JS to be run by this `Environment` afterwards?